libs/application/templates/passport-annulment/src/fields/DoneImage/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { Box } from '@island.is/island-ui/core'
+import School from './assets/School'
+
+export const DoneImage = () => {
+  return (
+    <Box display={'flex'} justifyContent={'center'} marginY={3}>
+      <School />
+    </Box>
+  )
+}
+
+export default DoneImage
```

Comment:
Strange to use School for passport?

Reply:
i need to double check which image we are going to use here, will come back with a replacement